docs/visual-basic/programming-guide/language-features/procedures/snippets/how-to-call-an-event-handler/Class1.vb

```diff
@@ -0,0 +1,26 @@
+    '****************************************************************************
```

Comment:
@adegeo, the code here is WinForms, but actually have no forms in it. Should I introduce the vbproj here as a console app, and reference System.Windows.Forms?

Reply:
Sure, though it may just be easier to create a winforms vb project and use that.